app/views/ansible_credential/new.html.haml

```diff
@@ -1,2 +1,2 @@
 #main_div
-  = render :partial => 'credential_form', :locals => {:newRecord => true}
+  = react('AnsibleCredentialsForm')
```

Comment:
Use the parentheses around this consistently with the other one, so either no parentheses here like above, or the opposite. Ruby doesn't need parentheses for function calls...but I care more about consistency :wink: